java/src/jmri/jmrix/dccpp/swing/DCCppRosterExportAction.java

```diff
@@ -0,0 +1,58 @@
+package jmri.jmrix.dccpp.swing;
+
+import java.awt.event.ActionEvent;
+import javax.swing.AbstractAction;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import javax.script.Bindings;
+import javax.script.ScriptException;
+import javax.script.SimpleBindings;
+
+import jmri.JmriException;
+import jmri.script.ScriptEngineSelector;
+
+/**
+ * Menu action to Export all JMRI Roster Entries as ROSTER() macro calls
+ *  This adds a menu item to the "DCC++" menu. Taking this option opens a 
+ *  window where the ROSTER() lines can be copied into your myAutomation.h 
+ *  Syntax: ROSTER(999,"Loco Name","F0/F1/*F2/F3/F4/F5/F6/F7/F8")
+ * NOTE: calls existing script for compatibility
```

Comment:
It would be good with a note in the script that this script is called by the `jmri.jmrix.dccpp.swing.DCCppRosterExportAction` action.

In case someone changes the script, it's good to know where it's used. For example, if someone later creates a "version 2.0" script that's a copy of the original script, an extra menu item might be added for that new script as well.